cerbero/packages/wix.py

```diff
@@ -270,13 +271,25 @@ def __init__(self, config, package):
             self.ui_type = 'WixUI_Minimal'
         else:
             self.ui_type = 'WixUI_Mondo'
+        self._init_replacements()
 
     def write(self, output_dir):
+        config_out_path = self.copy(output_dir)
+        self.replace(config_out_path)
+        return config_out_path
+
+    def copy(self, output_dir):
         config_out_path = os.path.join(output_dir,
                 os.path.basename(self.wix_config))
         shutil.copy(self.config_path, os.path.join(output_dir,
                     os.path.basename(self.wix_config)))
-        replacements = {
+        return config_out_path
+
+    def replace(self, config_out_path):
+        shell.replace(config_out_path, self.replacements)
```

Comment:
Although now that I'm looking at it, I probably don't need the two methods, as I already have the init_replacements method.